packages/flutter_image/lib/network.dart

```diff
@@ -95,10 +99,13 @@ class NetworkImageWithRetry extends ImageProvider<NetworkImageWithRetry> {
   }
 
   @override
-  // TODO(cyanglaz): migrate to use the new APIs
-  // https://github.com/flutter/flutter/issues/105336
-  // ignore: deprecated_member_use
-  ImageStreamCompleter load(NetworkImageWithRetry key, DecoderCallback decode) {
+  ImageStreamCompleter loadBuffer(
+    NetworkImageWithRetry key,
+    // TODO(cyanglaz): migrate to use the new APIs
+    // https://github.com/flutter/flutter/issues/105336
```

Comment:
This issue is about APIs deprecated a year ago, so shouldn't still be applicable to new code. If there's a new migration needed, there should be a new issue describing it clearly (the old API, the new API, and the version it's in).

Reply:
Created https://github.com/flutter/flutter/issues/132856